Day-01/go/base2code/day01.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	content, _ := os.ReadFile("day01/input.txt")
```

Comment:
Please place the input file in the same folder as the code

```suggestion
	content, _ := os.ReadFile("input.txt")
```